.github/workflows/deploy-gh-pages.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       with:
         workflow: build-ci-test.yaml
         github_token: ${{secrets.GITHUB_TOKEN}}
-        path: ./frontend/build/
+        path: ./frontend/
 
     - name: Deploy 🚀
       uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
Why are we using this? pnpm run deploy kaam nahi kar raha kya?

Reply:
nopes